Index: config.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config.py b/config.py
new file mode 100644
--- /dev/null	(date 1712578591263)
+++ b/config.py	(date 1712578591263)
@@ -0,0 +1,2 @@
+BOT_TOKEN = '6407545116:AAF_hetcu10dMsPR0rIStuNRqqXZoloptT4'
+OPEN_WEATHER_API_KEY = '8221c420232de1b659a9343bae8b37fa'
