Index: config.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config.py b/config.py
new file mode 100644
--- /dev/null	(date 1712388902908)
+++ b/config.py	(date 1712388902908)
@@ -0,0 +1,2 @@
+BOT_TOKEN = '7082243141:AAG6CZeU2w6Eq-hI4dERkvAQKjVaeCdVvQs'
+OPEN_WEATHER_API_KEY = 'b02696c83d542d25a004c9b8fe2d04d2'
